### PR TITLE
Add worker-src to CSP article.

### PR DESCRIPTION
**Pull request by jpmedley**
With a few other minor tweaks.

### CODE REVIEW

**Comment by jpmedley** [comment]
@ebidel @petele We're getting the component error because it's missing from the json file.  Blink>SecurityFeature>ContentSecurityPolicy [has tickets assigned to it](https://bugs.chromium.org/p/chromium/issues/list?q=component:Blink%3ESecurityFeature%3EContentSecurityPolicy).

**Comment by petele** [comment]
Would you please fix the conflicts, then happy to merge.

**Comment by jpmedley** [comment]
@petele I need somebody with expertise to review this. I sent this to Mike last week but have not heard from him.

**Comment by petele** [comment]
@jpmedley - would you please fix the conflicts ASAP

**Comment by ebidel** [comment]
@jpmedley can you use "Blink>SecurityFeature" for now? Sometimes bug owners use labels that aren't listed in https://storage.googleapis.com/chromium-owners/component_map.json (yet).

File: src/content/en/fundamentals/security/csp/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ book_path: /web/fundamentals/_book.yaml
 description: Content Security Policy can significantly reduce the risk and impact of cross-site scripting attacks in modern browsers.
 
 {# wf_published_on: 2012-06-15 #}
-{# wf_updated_on: 2016-02-19 #}
+{# wf_updated_on: 2017-08-11 #}
```
Comment (mikewest) [inline src/content/en/fundamentals/security/csp/index.md]: (I can't comment on the image, but we should totally update it at some point. :) )

File: src/content/en/fundamentals/security/csp/index.md [hunk 2]
```diff
@@ -104,7 +110,8 @@ fonts could be enabled via `font-src https://themes.googleusercontent.com`.
 This directive applies to `<frame>`, `<iframe>`, `<embed>`, and `<applet>` tags.
 This directive can't be used in `<meta>` tags and applies only to non-HTML
 resources.
-* **`frame-src`** deprecated. Use **`child-src`** instead.
+* **`frame-src`** was deprecated in level 2, but appears not to be in level 3.
+  Exactly how this will work is still behing hashed out.
```
Comment (mikewest) [inline src/content/en/fundamentals/security/csp/index.md]: We're done hashing. Item 2 of https://w3c.github.io/webappsec-csp/#changes-from-level-2 is what CSP3 is going to run with. Pinky swear.

File: src/content/en/fundamentals/security/csp/index.md [hunk 3]
```diff
@@ -116,6 +123,10 @@ tags.
 * **`upgrade-insecure-requests`** instructs user agents to rewrite URL schemes,
 changing HTTP to HTTPS. This directive is for websites with large numbers of
 old    URL's that need to be rewritten.
+* **`worker-src`** is a CSP Level 3 directive that restricts the URLs that may
+be loaded as a worker, shared worker, or service worker. As of July 2017, this
+directive has
+[limited implementations](https://www.chromestatus.com/features/5922594955984896).
```
Comment (mikewest) [inline src/content/en/fundamentals/security/csp/index.md]: SGTM!

File: src/content/en/fundamentals/security/csp/index.md [hunk 4]
```diff
@@ -454,14 +467,15 @@ best be able to support them within the protective confines of CSP.
...
+* Google's [+1 button](/+/web/+1button/)
 includes a script from `https://apis.google.com`, and embeds an `<iframe>` from
 `https://plusone.google.com`. You need a policy that includes both these
 origins in order to embed the button. A minimal policy would be `script-src
 https://apis.google.com; child-src https://plusone.google.com`. You also need
 to ensure that the snippet of JavaScript that Google provides is pulled out into
 an external JavaScript file. If you have an existing policy using `frame-src`,
-you need to change it to `child-src`.
+you were required to change it to `child-src` for CSP Level 2. This may change
+in CSP Level 3.
```
Comment (mikewest) [inline src/content/en/fundamentals/security/csp/index.md]: s/may/did/

Sorry. It's a mess, and mostly my fault. :(

**Pull request merged**